pubspec.yaml

```diff
@@ -22,6 +22,7 @@ dependencies:
 
   analyzer: ^0.39.4
   black_hole_flutter: ^0.2.3
```

Comment:
Can you please update <kbd>black_hole_flutter</kbd> to v0.2.4? It contains a fix for the extended `FanyFab` not working on `EditSubmissionScreeen` without a submission.